Add PR template (#2299)
python-telegram-bot/python-telegram-bot: .github/pull_request_template.md

-->

### Checklist for PRs

- [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to user facing changes
- [ ] Created new or adapted existing unit tests
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)

### If the PR contains API changes (delete if not relevant)

* New classes:
    - [ ] Added `self._id_attrs` and corresponding documentation
    - [ ] `__init__` accepts `**_kwargs`
    
* Added new shortcuts:
    - [ ] At `Chat` & `User` for all methods that accept `chat/user_id`
    - [ ] At `Message` for all methods that accept `chat_id` and `message_id`
    - [ ] For new `Message` shortcuts: Added `quote` argument if methods accepts `reply_to_message_id`
    - [ ] At `CallbackQuery` for all methods that accept either `chat_id` and `message_id` or `inline_message_id`
    
* If relevant:

    - [ ] Added new constants at `telegram.constants` and shortcuts to them as class variables
    - [ ] Added new handlers for new update types

>     - [ ] Added new filters for new message (sub)types

Reviewer: Another point after that - 

- Added or updated documentation for the changed class(es) and/or method(s)

<!--

> Hey! You're PRing? Cool! Please have a look at the below checklist. It's here to help both you and the maintainers remember some things. For more details, please see our contribution guide.

Reviewer 2: ```suggestion
Hey! You're PRing? Cool! Please have a look at the below checklist. It's here to help both you and the maintainers to remember some aspects. Make sure to check out our contribution guide (https://github.com/python-telegram-bot/python-telegram-bot/blob/master/.github/CONTRIBUTING.rst).
```

-->

### Checklist for PRs

> - [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` if appropriate 

Reviewer 2: ```suggestion
- [ ] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to user facing changes
```

- [ ] Created new or adapted existing unit tests

> - [ ] Added myself to `AUTHORS.rst` (optional)

Reviewer 2: ```suggestion
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)
```

> ### If the PR contains API changes (delete if not relevant)

Reviewer 2: ```suggestion
### When the PR contains API changes (delete if not relevant)
```

Author: When = time, If = conditional ?!

Reviewer 2: yeah but you already have the condition in the parenthesis. Also, you can use when to introduce something that is going to happen/is likely to happen. It is closely bound to being in the future but that is also true since the future happens after the PR person reads the line :)

Reviewer 2: ```suggestion
### If the PR contains API changes (otherwise, you can delete this passage)
```

alternative

Author: > yeah but you already have the condition in the parenthesis. Also, you can use when to introduce something that is going to happen/is likely to happen. It is closely bound to being in the future but that is also true since the future happens after the PR person reads the line :)

Reading this feels a bit like whatching Interstellar ;P I'll go with the second suggestion :D

* New classes:
    - [ ] Added `self._id_attrs` and corresponding documentation
    - [ ] `__init__` accepts `**_kwargs`
    

> * Added new shortcuts:
> 
>     - [ ] At `Chat` & `User` for all methods that accept `chat/user_ad`

Reviewer 2: ```suggestion
* Added new shortcuts:
    - [ ] At `Chat` & `User` for all methods that accept `chat/user_id`
```

    - [ ] At `Message` for all methods that accept `chat_id` and `message_id`
    - [ ] For new `Message` shortcuts: Added `quote` argument if methods accepts `reply_to_message_id`

>     - [ ] At `CallbackQuery` for all methods that accept either `chat_id` and `message_id` or `inline_message_id`

Reviewer 2: I'd probably change `At` to `In`?

Author: mh … maybe … don't care enough to make a commit for it :D
